### ISSUE TITLE
Metal does not accept viewport larger than screen

### ISSUE THREAD

**Issue by kpeeters** [google/filament#5892]
Since https://github.com/google/filament/pull/5710/commits/9000c255d09a6b9c5d48fa5265af05bc5979fb83 the Metal backend produces a black screen when the viewport is larger than the screen/framebuffer. Is that intentional? The OpenGL and Vulkan backends have no problem with this.

**Comment by bejado** [comment]
No, not intentional. Can you see if https://github.com/google/filament/pull/5896 fixes the issue for you?

**Comment by kpeeters** [comment]
That commit https://github.com/google/filament/pull/5896 is based on `HEAD`, which for me suffers from https://github.com/google/filament/issues/5891, even with #5896 applied. Applying the diff to https://github.com/google/filament/commit/9000c255d09a6b9c5d48fa5265af05bc5979fb83 (where the problem described in the current issue first arises) leads to non-trivial conflicts.
